assets/src/edit-story/components/library/panes/shapes/shapesPane.js

```diff
@@ -71,29 +74,32 @@ function ShapesPane(props) {
       <Section title={__('Basic shapes', 'web-stories')}>
         <SectionContent>
           {/** Basic masks */}
-          {MASKS.map((mask) => (
-            <ShapePreview
-              key={mask.type}
-              onClick={() => {
-                insertElement('shape', {
-                  backgroundColor: createSolid(51, 51, 51),
-                  width: DEFAULT_ELEMENT_WIDTH,
-                  mask: {
-                    type: mask.type,
-                  },
-                });
-              }}
-              alt={mask.name}
-            >
-              <svg
-                viewBox={'0 0 1 1'}
-                width={PREVIEW_SIZE}
-                height={PREVIEW_SIZE}
+          {MASKS.map((mask) => {
+            return (
+              <ShapePreview
+                key={mask.type}
+                onClick={() => {
+                  insertElement('shape', {
+                    backgroundColor: createSolid(51, 51, 51),
+                    width: DEFAULT_ELEMENT_WIDTH,
+                    height: DEFAULT_ELEMENT_WIDTH * mask.ratio,
+                    mask: {
+                      type: mask.type,
+                    },
+                  });
+                }}
+                alt={mask.name}
               >
-                <Path d={mask.path} />
-              </svg>
-            </ShapePreview>
-          ))}
+                <svg
+                  viewBox={`0 0 1 ${mask.ratio}`}
+                  width={PREVIEW_SIZE}
+                  height={PREVIEW_SIZE * mask.ratio}
```

Comment:
To reclarify: it's totally ok to do `height=PREVIEW_SIZE` and `width=ratioed(PREVIEW_SIZE)`

Reply:
fixed.